File: app/src/main/java/com/mfrido/aplikasimendatakeluarga/InformasiPemilu.java
package com.mfrido.aplikasimendatakeluarga;

import androidx.appcompat.app.AppCompatActivity;

import android.os.Bundle;
import android.webkit.WebView;

public class InformasiPemilu extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_informasi_pemilu);
        WebView webView = findViewById(R.id.webViewPemilu);
        webView.getSettings().setJavaScriptEnabled(true);
        webView.loadUrl("https://infopemilu.kpu.go.id/Pemilu/Peserta_pemilu");
    }
}

File: app/src/main/java/com/mfrido/aplikasimendatakeluarga/InformasiPemilu.kt
package com.mfrido.aplikasimendatakeluarga

import android.os.Bundle
import android.webkit.WebView
import androidx.appcompat.app.AppCompatActivity

class InformasiPemilu : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_informasi_pemilu)
        val webView = findViewById<WebView>(R.id.webViewPemilu)
        webView.loadUrl("put link here")
    }
}